src/ros_filter.cpp

```diff
@@ -834,18 +834,18 @@ void RosFilter<T>::loadParams()
   // Transform future dating
   double offset_tmp = this->declare_parameter("transform_time_offset", 0.0);
   tf_time_offset_ =
-    rclcpp::Duration(filter_utilities::secToNanosec(offset_tmp));
+    rclcpp::Duration::from_nanoseconds(filter_utilities::secToNanosec(offset_tmp));
```

Comment:
this and the 2 under are converting from s -> ns -> duration. You could remove the `filter_utilities::secToNanosec` call and replace it with just `from_seconds`

Reply:
Of course :upside_down_face: 
See ca9e807